easybuild/easyblocks/s/scipy.py

```diff
@@ -42,7 +42,7 @@
 from easybuild.easyblocks.generic.mesonninja import MesonNinja
 from easybuild.easyblocks.generic.pythonpackage import PythonPackage, det_pylibdir
 from easybuild.framework.easyconfig import CUSTOM
-from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.build_log import EasyBuildError, print_warning
 from easybuild.tools.filetools import change_dir, copy_dir
```

Comment:
```suggestion
from easybuild.tools.filetools import change_dir, copy_dir, copy_file
```

Reply:
fixed in [de55fd4](https://github.com/easybuilders/easybuild-easyblocks/pull/2903/commits/de55fd4f28aaf79b3eec21c6c433e592c79c8219)